exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExporterWorker.java

```diff
@@ -60,7 +60,8 @@
 
   private static final Logger logger = Logger.getLogger(StackdriverExporterWorker.class.getName());
 
-  @VisibleForTesting static final int MAX_BATCH_EXPORT_SIZE = 3;
+  // Stackdriver Monitoring v3 only accepts up to 200 TimeSeries per CreateTimeSeries call.
+  @VisibleForTesting static final int MAX_BATCH_EXPORT_SIZE = 200;
```

Comment:
Is there a StackDriver constant that we could use?

Reply:
Didn't find one (not even in Stackdriver's documentation). I think they handle it in their backend.